### DISCUSSION TITLE
Can't get html-based description working with Outlook

### DISCUSSION THREAD

**Discussion by pgoldweic** [ical4j/ical4j#785] [General]
I've followed the wiki's instructions to add an Oulook-only HTML description to my calendar event, but Outlook (2016) appears to ignore it. Is there a particular version of Outlook this has been tested with, or is it supposed to work with? 
I have created an all day event (also following examples online) with a regular description, and also the HTML one.
Please advise. Thanks.
Additional question: , is Outlook the only email client that allows -in theory- html descriptions?

**Comment by pgoldweic** [comment]
I have just found the answer to my first question: yes, indeed, the recipe for Outlook does *not* work in Outlook 2016, and it has been considered 'not a bug' by Microsoft experts (see https://answers.microsoft.com/en-us/msoffice/forum/msoffice_outlook-mso_win10-mso_o365b/outlook-2016-ics-files-fail-to-show-html/3bce904b-4f6e-4c80-b94e-95cb7e06f493 among other similar pages). They explain that the 'work around' is to NOT include a regular description, which makes the HTML description show (not a great solution, obviously, if one would like compatibility with other email clients).
I think this deserves to be added to the wiki, not to create false expectations about this functionality.

I'm still curious about my next question though: is there a way to get an HTML description to show for email clients other than Outlook?

**Comment by douglm** [comment]
This may be relevant:

A new version of I-D, draft-ietf-calext-eventpub-extensions-09.txt
has been successfully submitted by Michael Douglass and posted to the
IETF repository.

Name:		draft-ietf-calext-eventpub-extensions
Revision:	09
Title:		Event Publishing Extensions to iCalendar
Document date:	2018-08-30
Group:		calext
Pages:		32
URL:https://www.ietf.org/internet-drafts/draft-ietf-calext-eventpub-extensions-09.txt
Status:https://datatracker.ietf.org/doc/draft-ietf-calext-eventpub-extensions/
Htmlized:https://tools.ietf.org/html/draft-ietf-calext-eventpub-extensions-09
Htmlized:https://datatracker.ietf.org/doc/html/draft-ietf-calext-eventpub-extensions
Diff:https://www.ietf.org/rfcdiff?url2=draft-ietf-calext-eventpub-extensions-09

Abstract:
    This specification updates [RFC5545] by introducing a number of new
    iCalendar properties and components which are of particular use for
    event publishers and in social networking.

    This specification also defines a new STRUCTURED-DATA property for
    iCalendar [RFC5545] to allow for data that is directly pertinent to
    an event or task to be included with the calendar data.

**Comment by pgoldweic** [comment]
Great to hear about this specification update, and certainly hoping that it gets implemented soon!

**Comment by benfortuna** [comment]
https://tools.ietf.org/html/draft-ietf-calext-eventpub-extensions-18
